containers/python/rpc.py

```diff
@@ -312,13 +311,25 @@ def run(self, metric_conn):
 
                         response.send(socket, self.event_history)
 
-                        pred_metric['model_pred_count'] += 1
-
-                        metric_conn.send(pred_metric)
+                        recv_time = (t2 - t1).microseconds
+                        parse_time = (t3 - t2).microseconds
+                        handle_time = (t4 - t3).microseconds
+
+                        model_container_metric = {}
+                        model_container_metric['pred_total'] = 1
+                        model_container_metric[
+                            'recv_time_ms'] = recv_time / 1000
+                        model_container_metric[
+                            'parse_time_ms'] = parse_time / 1000
+                        model_container_metric[
+                            'handle_time_ms'] = handle_time / 1000
+                        model_container_metric['end_to_end_latency_ms'] = (
+                            recv_time + parse_time + handle_time) / 1000
```

Comment:
Can you make these divisions by `1000.0`?

Reply:
Fixed.